### PR TITLE
Docusaurus Snyk fixes

### PR DESCRIPTION
**Pull request by bhazuka**
## Summary of Changes

Addresses issues related to Snyk CVE issues found with current versions. Upgraded to use latest version of Docusaurus to attempt to resolve.

See [ORCA-436](https://bugs.earthdata.nasa.gov/browse/ORCA-436) for details.

## Changes

* Updated the following files
  * package.json
  * package-lock.json
* Updated documentation to dispay correct npm versions to use

## Type of change

Please delete options that are not relevant.

- [X] Bug fix (non-breaking change which fixes an issue)
- [X] This change requires a documentation update

## How Has This Been Tested?

Ran documentation in development mode and verified page renderings manually. Rad build and verified no errors occurred. 

## Checklist:

- [X] I have performed a self-review of my own code
- [X] I have made corresponding changes to the necessary documentation (remove those that do not apply)
    - [X] CHANGELOG.md
    - [X] Development documentation
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
    - [X] Manual tests (outlined above)
- [X] My code has passed security scanning
    - [X] Snyk
    - [X] git-secrets

## What is the current behavior?

See Snyk report https://app.snyk.io/org/lp-daac/project/30d51baa-7ef9-4c11-a4c6-a7a71e85a4e0

### CODE REVIEW

**Review by rizbihassan — APPROVED** [review]

File: website/docs/developer/development-guide/documentation/documentation-dev-env.md [hunk 1]
```diff
@@ -19,13 +19,13 @@ The installation steps below goes into further details.
...
+   curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.1/install.sh | bash
    ```
 
-2. Install the proper Node.js and npm versions to your machine. Currently Node v14.17.6 should be used.
+2. Install the proper Node.js and npm versions to your machine. Currently Node v16.15.1 should be used.
    ```sh
-   nvm install v16.13.1
-   nvm use v16.13.1
+   nvm install --lts
+   nvm use --lts
```
Comment (AndrewDorn) [inline website/docs/developer/development-guide/documentation/documentation-dev-env.md:28]: Auto-updating makes me a bit nervous, but I understand why it is desired from a security standpoint.

File: website/docs/developer/development-guide/documentation/documentation-dev-env.md [hunk 2]
```diff
@@ -19,13 +19,13 @@ The installation steps below goes into further details.
 
 1. Install the latest [Node Version Manager (nvm)](https://github.com/nvm-sh/nvm) application to manage Node.js and npm versions.
    ```sh
-   curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.0/install.sh | bash
+   curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.1/install.sh | bash
    ```
 
-2. Install the proper Node.js and npm versions to your machine. Currently Node v14.17.6 should be used.
+2. Install the proper Node.js and npm versions to your machine. Currently Node v16.15.1 should be used.
```
Comment (AndrewDorn) [inline website/docs/developer/development-guide/documentation/documentation-dev-env.md:25]: This comment conflicts with the `lts` version below.

File: README.md [hunk 3]
```diff
@@ -8,8 +8,8 @@ The documentation is available for developers, data managers, and users.
 Additional documentation is being added continually.
 
 Make sure you are using the following node.js versions to view the documentation.
-- npm 6.14.10
-- node 12.15.0
+- npm 8.11.0
+- node 16.15.1
```
Comment (AndrewDorn) [inline README.md:12]: This, or lts?

**Review by AndrewDorn — CHANGES_REQUESTED** [review]
Couple comments about versions. If we're just taking the latest version, we might need to adjust the docs away from specific versions.

File: website/docs/developer/development-guide/documentation/documentation-dev-env.md (hunk 1 above)
Comment (bhazuka) [reply website/docs/developer/development-guide/documentation/documentation-dev-env.md]: I was being a little lazy here but you make a good point. 16.15.1 is the latest LTS and there is the possibility the LTS 18 would come out and could cause issues.I did a little reading and we could talk about limiting it to the latest version within the LTS major version so v16.x.x using the `--lts=Gallium` flag. For the meantime I reverted it to use the specific version to keep everything consistent throughout.

File: website/docs/developer/development-guide/documentation/documentation-dev-env.md (hunk 2 above)
Comment (bhazuka) [reply website/docs/developer/development-guide/documentation/documentation-dev-env.md:25]: Reverted to use specific version in command.

**Review by AndrewDorn — APPROVED** [review]

**Pull request merged**